src/main/java/org/betonquest/betonquest/events/ObjectiveEvent.java

```diff
@@ -24,39 +25,53 @@
 @CustomLog
 public class ObjectiveEvent extends QuestEvent {
 
-    private final ObjectiveID objective;
+    /**
+     * The BetonQuest instance.
+     */
+    public static final BetonQuest BETONQUEST = BetonQuest.getInstance();
```

Comment:
Thsi will break: The BetonQuest instance is only available after Bukkit has initialized the plugin, but the static variable might be initialized before, effectively setting the value to `null`. Use `BetonQuest.getInstance()` either in the constructor to set a field (class variable) or use it directly in the code.

Reply:
You mean that a faulty (?) class loader could load the class, therefore initializing the `BETONQUEST` variable, before Bukkit initialized the BetonQuest main class?